automated-comments/java/hamming/must_calculate_hamming_distance.md

```diff
@@ -0,0 +1,3 @@
+There are no loops or lambdas in this solution.
```

Comment:
```suggestion
```

Reply:
Could be removed as this is more of an explanation what the analyzer is checking for.